index.js

```diff
@@ -53,6 +53,8 @@ module.exports = app => {
       name: core.getInput('name') || undefined
     })
 
+    const shouldDraft = core.getInput('publish').toLowerCase() !== 'true'
+
     let createOrUpdateReleaseResponse
     if (!draftRelease) {
       log({ app, context, message: 'Creating new draft release' })

```

Comment:
Just noticed this when testing this branch in the workflow.
Even when release is published (`shouldDraft` is false), the logs say `Creating new draft release` or `Updating existing draft release` which not true when release is published, not drafted.

Reply:
Good catch. @toolmantim/@TimonVS if you're interested in eventually merging this PR, do you have a preference for how to update the language here?

Reply:
(When resolving the conflict with latest `master`, I just removed the world "draft" from the message, but I'm happy to make other updates if there's a preference to get more detail in there)